File: src/codes/Users_Manual/modulation/wav_1.py
~import pathlib
~from pyautd3 import kHz
~from pyautd3.modulation import SincInterpolation, BlackMan
~from pyautd3.modulation.audio_file import Wav
~path = pathlib.Path("path/to/foo.wav")
m = Wav.new_with_resample(path, 4 * kHz, SincInterpolation(BlackMan(32)))

File: src/codes/Users_Manual/modulation/wav_1.rs
# use std::num::NonZeroUsize;
# use autd3::prelude::*;
# use autd3::modulation::resampler::{SincInterpolation, Blackman};
# use autd3_modulation_audio_file::Wav;
# #[allow(unused_variables)]
# fn main() -> Result<(), Box<dyn std::error::Error>> {
# let path = "path/to/foo.wav";
let m = Wav::new_with_resample(
    path,
    4 * kHz,
    SincInterpolation {
        window: Blackman {
            size: NonZeroUsize::new(32).unwrap(),
        },
    },
)?;
# Ok(())
# }